var/spack/repos/builtin/packages/aotriton/package.py

```diff
@@ -0,0 +1,80 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class Aotriton(CMakePackage):
+    """Ahead of Time (AOT) Triton Math Library."""
+
+    homepage = "https://github.com/ROCm/aotriton"
+    url = "https://github.com/ROCm/aotriton/archive/refs/tags/0.8.2b.tar.gz"
+
+    maintainers("afzpatel", "srekolam", "renjithravindrankannath")
+
+    license("MIT")
+
+    version(
+        "0.8.2b",
+        git="https://github.com/ROCm/aotriton",
+        tag="0.8.2b",
+        commit="b24f43a9771622faa157155568b9a200c3b49e41",
+        submodules=True,
+    )
+    version(
+        "0.8.1b",
+        git="https://github.com/ROCm/aotriton",
+        tag="0.8.1b",
+        commit="3a80554a88ae3b1bcf4b27bc74ad9d7b913b58f6",
+        submodules=True,
+    )
+    version(
+        "0.8b",
+        git="https://github.com/ROCm/aotriton",
+        tag="0.8b",
+        commit="6f8cbcac8a92775291bb1ba8f514d4beb350baf4",
+        submodules=True,
+    )
+
+    generator("ninja")
+    depends_on("c", type="build")  # generated
+    depends_on("cxx", type="build")  # generated
+
+    depends_on("py-setuptools@40.8:", type="build")
+    depends_on("py-filelock", type=("build", "run"))
```

Comment:
Any other dependencies in https://github.com/ROCm/aotriton/blob/main/requirements.txt needed?